demos/wang-2018/src/convert_to_csv.py

```diff
@@ -0,0 +1,107 @@
+import argparse
+import json
+import pandas as pd
+
+
+def convert_to_csv(args):
+    with open(args.input_cells) as f:
+        cells_json = json.load(f)
+    with open(args.input_molecules) as f:
+        molecules_json = json.load(f)
+    with open(args.input_expression_matrix) as f:
+        expression_matrix = json.load(f)
+
+    molecules_df = pd.DataFrame(index=[], columns=["X", "Y", "Gene"])
+    start_at = 0
+    for molecule_type, molecule_arr in molecules_json.items():
+        molecule_type_ids = [str(i) for i in range(start_at, start_at + len(molecule_arr))]
+        start_at += len(molecule_arr)
+        molecule_type_df = pd.DataFrame(index=molecule_type_ids, columns=molecules_df.columns.values.tolist())
+        molecule_type_df['Gene'] = molecule_type
+        molecule_type_df['X'] = [m[0] for m in molecule_arr]
+        molecule_type_df['Y'] = [m[1] for m in molecule_arr]
```

Comment:
This looks repeated (but maybe not?) but I feel less strongly about this going in the core - could be good to just make a util here.